ddb_cp.module

```diff
@@ -430,11 +449,13 @@ function _ddb_cp_get_test_result($site = NULL) {
 /**
  * Returns an HTML formatted test status message string.
  *
- * @param $site
+ * @param string $site
  *   'dev' or 'stg' site to format message for.
- * @param $status
+ * @param mixed $status
  *   Site-specific status record as selected by ddb_cp_ajax_status()
  *
+ * @todo: All the HTML in this would be nice to have into tpl's and theme functions.
```

Comment:
I have talk with Gitte about this and have agree to create an issue on it and accept it as is for now.

See issue at http://platform.dandigbib.org/issues/787